### PR TITLE
AP-2142: Implement ProceedingMeritsTask::AttemptsToSettleForm

### PR DESCRIPTION
**Pull request by colinbruce**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-2142)

This PR moves the AttemptsToSettle model as it hadn't been namespaced to match the other ProceedingMeritsTasks
It creates a Form, Controller and views for the recording of the Attempt To Settle for each Proceeding Type

This has been rebased onto master containing the Merits Task List

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/forms/providers/proceeding_merits_task/attempts_to_settle_form.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module Providers
+  module ProceedingMeritsTask
+    class AttemptsToSettleForm
+      include BaseForm
+
+      form_for ::ProceedingMeritsTask::AttemptsToSettle
+
+      attr_accessor :attempts_made, :application_proceeding_type_id
```
Comment (Obsiye) [inline app/forms/providers/proceeding_merits_task/attempts_to_settle_form.rb:8]: just wondering why you have :application_proceeding_type_id as an attr_accessor?

Comment (Obsiye) [inline app/forms/providers/proceeding_merits_task/attempts_to_settle_form.rb:8]: Just wondering why you have :application_proceeding_type_id as an accessor ?

Comment (colinbruce) [reply app/forms/providers/proceeding_merits_task/attempts_to_settle_form.rb]: Nice catch, as discussed this has been removed and is no longer needed 👍

**Review by Obsiye — APPROVED** [review]
LGTM!

**Pull request merged**